docs/source/conf.py

```diff
@@ -4,16 +4,6 @@
 # list see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
```

Comment:
suggest leave the path setup comments - it's from the boilerplate code

Reply:
But it points to the local path without the *so file

Reply:
just the boilerplate comment